hack/bootstrap-ginkgo.sh

```diff
@@ -4,6 +4,8 @@ set -e
 
 source $(dirname "$0")/common.sh
 
+(cd ${KUBEVIRT_DIR} && go install github.com/onsi/ginkgo/v2/ginkgo)
```

Comment:
I am not sure when this script is called, but it is odd that it has no fixed release.
We are not interested in bumping it without control.

It will be nice if this can be explained in more details, so we can learn why other unit tests work and some do not (I refer to the fixes presented below).

Reply:
See the note in https://onsi.github.io/ginkgo/#installing-ginkgo
This ensures 1:1 with go.mod

Reply:
Maybe we are not in sync here.
I do not understand why you had to add this installation here.

Reply:
So we don't have the ginkgo at all at the moment,
so we have to install it somehow. I chose here instead of the Dockerfile of the builder
because that way we are ensured 1:1 versioning with go.mod ginkgo

Reply:
Sure we have it, this is why all the other unit tests are running.
You are now claiming that some specific unit tests are not running at all, and that is why you  had to add it.
There is a contradiction here that I do not understand.

Reply:
So unless I am hallucinating, the fuzz pkg is not ran as part of `make test` today.
I made it fail and make test was green:
```bash
# make test
...
14:43:17: Executed 132 out of 132 tests: 132 tests pass.
# go clean -testcache && go test -v ./pkg/virt-api/webhooks/fuzz
=== RUN   FuzzAdmitter
--- FAIL: FuzzAdmitter (0.00s)
panic: ...
```

Reply:
NVM. it runs as part of `make fuzz`

Reply:
To chime in here, I think this script should just use the ginkgo binary from the `_out` folder, as [`hack/functest.sh` does](https://github.com/kubevirt/kubevirt/blob/main/hack/functests.sh#L69) 

FYI: built here: https://github.com/kubevirt/kubevirt/blob/main/BUILD.bazel#L521

Reply:
I see, but the problem is this script is ran as part of the `make generate` target (not functest)

Reply:
So adding a dependency to `bazel-build-functests` in `generate` make target would fix that, no?

Reply:
Hmm this script targets unit tests so I am not sure about that

Reply:
I could add a dependency on building ginkgo on both of these make targets, WDYT @dhiller?

Reply:
Makes sense.

Reply:
@dhiller PTAL :pray: